central/authprovider/service/service_impl.go

```diff
@@ -183,6 +188,10 @@ func (s *serviceImpl) PutAuthProvider(ctx context.Context, request *storage.Auth
 		return nil, errox.NotFound.Newf("auth provider with id %q does not exist", request.GetId())
 	}
 
+	// Use the stored provider's required attributes values, if there are any, during creation. This way we
+	// can skip checks regarding required attributes and deliberately ignore values send from UI.
+	storedProvider := provider.StorageView()
```

Comment:
I checked via API to create a auth provider:
```bash
curl --location --request POST 'https://localhost:8000/v1/authProviders' \
--header 'Authorization: Basic <redacted>' \
--header 'Content-Type: application/json' \
--data-raw '{
    "id": "",
    "name": "Test",
    "type": "auth0",
    "uiEndpoint": "localhost:8000",
    "enabled": true,
    "config": {
        "client_id": "<redacted>",
        "issuer": "<redacted>",
        "mode": "fragment"
    },
        "requiredAttributes": [
        {
            "attributeKey": "name",
            "attributeValue": "haus"
        }
    ]
}'
```

This will leave the created auth provider with `active: false`. Via `PUT` I'm able to make changes.
 _After_ logging in with the specific provider for the first time, the provider will be set active. 
So there would be a window between creation + first login where this could potentially be changed.

For Managed Services a user would have to login with RH SSO provider at least once (which, in turn, would mark it as active, thus never being able to change it). 

So I'm fine with deleting this for now.